autoware_msg_bag_converter/dependency.repos

```diff
@@ -0,0 +1,14 @@
+repositories:
+  # core
+  core/autoware_msgs:
+    type: git
+    url: https://github.com/autowarefoundation/autoware_msgs.git
+    version: main
+  core/autoware_common:
+    type: git
+    url: https://github.com/autowarefoundation/autoware_common.git
+    version: main
+  core/external/autoware_auto_msgs: # TODO(mfc): Remove when autoware_msgs is merged
```

Comment:
I think it is okay to delete the comment as it seems unnecessary.

Reply:
I fixed with the following commit hash
4617d5e081db6686f516894ff61f41f31088ba1d